packages/client/src/hooks/useChessboard.ts

```diff
@@ -144,10 +144,16 @@ const useChessboard = () => {
     return decodeHeroFn(inventory || []);
   }, [inventory, decodeHeroFn]);
 
-  const HeroTable = useEntityQuery([Has(Hero)]).map((row) => ({
-    ...getComponentValueStrict(Hero, row),
-    key: row,
-  }));
+  const HeroTable = useEntityQuery([Has(Hero)])?.map((row) => {
```

Comment:
This variable can be deleted since you can directly get value by using `getComponentValue`